coffea/hist/hist_tools.py

```diff
@@ -1318,3 +1318,81 @@ def identifiers(self, axis, overflow='none'):
             return out
         elif isinstance(axis, DenseAxis):
             return axis.identifiers(overflow=overflow)
+
+    def to_boost(self):
+        """Convert this coffea Hist object to a boost_histogram obbject
+
+        """
+        import boost_histogram
+
+        newaxes = []
+        for axis in self.axes():
+            if isinstance(axis, Bin) and axis._uniform:
+                newaxis = boost_histogram.axis.Regular(
+                    axis._bins,
+                    axis._lo,
+                    axis._hi,
+                    underflow=True,
+                    overflow=True,
+                )
+                newaxis.name = axis.name
+                newaxis.label = axis.label
+                newaxes.append(newaxis)
+            elif isinstance(axis, Bin) and not axis._uniform:
+                newaxis = boost_histogram.axis.Variable(
+                    axis.edges(),
+                    underflow=True,
+                    overflow=True,
+                )
+                newaxis.name = axis.name
+                newaxis.label = axis.label
+                newaxes.append(newaxis)
+            elif isinstance(axis, Cat):
+                identifiers = self.identifiers(axis)
+                newaxis = boost_histogram.axis.StrCategory(
+                    [x.name for x in identifiers],
+                )
+                newaxis.name = axis.name
+                newaxis.label = axis.label
+                newaxis.bin_labels = [x.label for x in identifiers]
```

Comment:
@andrzejnovak do you know if category custom labels (e.g. latex) are supported in hist/mplhep?

Reply:
mpl will render that fine, but I don't think hist HTML repr can interpret latex (@henrii)

Reply:
![image](https://user-images.githubusercontent.com/13226500/109125467-645d7b80-774c-11eb-9989-f7078057f113.png)

Reply:
I noticed that issue as well, but here I'm asking about `StrCategory` axes. In coffea hists, when displaying a histogram with a `StrCategory` axis, by default it would use the category names as bin labels, but one could optionally override the category names with custom labels (so that the filling code could still use a simple ascii shorthand like `jetptcut` while the plot would show a more expressive label like `$p_{T,j} \geq 100 GeV$`)